graphql/patch_resolver.go

```diff
@@ -103,6 +105,49 @@ func (r *patchResolver) Duration(ctx context.Context, obj *restModel.APIPatch) (
 	return makePatchDuration(timeTaken, makespan), nil
 }
 
+// GeneratedTaskCounts is the resolver for the generatedTaskCounts field.
+func (r *patchResolver) GeneratedTaskCounts(ctx context.Context, obj *restModel.APIPatch) (map[string]interface{}, error) {
+	patchID := utility.FromStringPtr(obj.Id)
+	p, err := patch.FindOneId(patchID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while finding patch with id: `%s`: %s", patchID, err.Error()))
+	}
+	if p == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Patch does not exist: `%s`", patchID))
+	}
```

Comment:
nit. formatting and regular quotes
```suggestion
	if err != nil {
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding patch with id '%s': %s", patchID, err.Error()))
	}
	if p == nil {
		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("patch '%s' does not exist", patchID))
	}
```